sdks/go/pkg/beam/core/typex/class_test.go

```diff
@@ -105,7 +105,7 @@ func TestClassOf(t *testing.T) {
 		{reflect.TypeOf(uintptr(0)), Invalid},          // uintptr
 		{reflect.TypeOf(func() {}), Invalid},           // function
 		{reflect.TypeOf(make(chan int)), Invalid},      // chan
-		{reflect.TypeOf(struct{ A error }{}), Invalid}, // public interface field
+		{reflect.TypeOf(struct{ An error }{}), Invalid}, // public interface field
```

Comment:
This one is code, not English, so it should probably have been excluded from linting.

Reply:
this ‘A’ is a variable